api/proto/teleport/integration/v1/awsoidc_service.proto

```diff
@@ -307,3 +313,42 @@ message ListEC2Response {
   // If non-empty, it can be used to request the next page.
   string next_token = 2;
 }
+
+// ListEKSClustersRequest is a request for a paginated list of AWS EKS Clusters.
+message ListEKSClustersRequest {
+  // Integration is the AWS OIDC Integration name.
+  // Required.
+  string integration = 1;
+  // Region is the AWS Region
+  // Required.
+  string region = 2;
+  // NextToken is the token to be used to fetch the next page.
+  // If empty, the first page is fetched.
+  string next_token = 3;
+}
+
+// EKSCluster represents an Amazon EKS Cluster.
+message EKSCluster {
+  // Name is the name of AWS EKS cluster.
+  string name = 1;
+  // Region is an AWS region.
+  string region = 2;
+  // Arn is an AWS ARN identification of the EKS cluster.
+  string arn = 3;
+  // Labels are labels of a EKS cluster.
+  map<string, string> labels = 4;
+  // JoinLabels are Teleport labels that should be injected into kube agent
+  // if the cluster will be enrolled into Teleport (agent installed on it).
+  map<string, string> join_labels = 5;
+  // Status is a current status of an EKS cluster in AWS.
+  string status = 6;
```

Comment:
status is a enum, can we make it a enum or identify which possible values are we receiving